src/theory/fp/theory_fp.cpp

```diff
@@ -411,7 +419,13 @@ void TheoryFp::convertAndEquateTerm(TNode node) {
 #endif
 
     } else {
-      Assert((node.getKind() == kind::EQUAL));
+      // Component bits should not be altered.
+      // These are the only bits that should be allowed through
```

Comment:
Is this a typo? Else, what does this sentence mean?

Reply:
Looking at it again, the extra cases are not needed so they can be removed, along with the comment.